src/plugins/shaders/src/components/effect_shader.rs

```diff
@@ -0,0 +1,117 @@
+use crate::traits::insert_effect_shader::InsertEffectShader;
+use bevy::{ecs::system::EntityCommands, prelude::*};
+use common::traits::push::Push;
+
+#[cfg(test)]
+use bevy::asset::UntypedAssetId;
+
+#[derive(Component, Default)]
+pub struct EffectShaders {
+	pub(crate) meshes: Vec<Handle<Mesh>>,
```

Comment:
add discussion about number of meshes (one vs many)

Reply:
Added issue about this https://github.com/codaishin/project-zyheeda-bevy/issues/228